code/Web Assigment 1/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Web Assignment 1</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script src="script.js" defer></script>
+  </head>
+  <body>
+    <div class="center-container">
+    <h1 id="banner">Light Mode</h1>
+    <button id="mode-btn">switch</button>

```

Comment:
This value probably wants to be "Dark" to match what happens later (but it's not a big deal)